pyproject.toml

```
@@ -1,11 +1,79 @@
 [build-system]
 requires = [
-    "setuptools>=42",
+    "setuptools @ git+https://github.com/abravalheri/setuptools@support-pyproject",
     "wheel",
     "setuptools_scm[toml]>=3.4.3"
 ]
 build-backend = "setuptools.build_meta"
 
+
+[project]
+name = "plumbum"
+description = "Plumbum: shell combinators library"
+readme = "README.rst"
+author = { name="Tomer Filiba", email="tomerfiliba@gmail.com" }
```

I think the standard mandates an array here (and also `authors` instead of `author`):

```suggestion
authors = [{ name="Tomer Filiba", email="tomerfiliba@gmail.com" }]
```